multiqc/plots/linegraph.py

```diff
@@ -222,25 +222,31 @@ def plot(data, pconfig=None):
             d.setdefault("color", scale.get_colour(di, lighten=1))
 
     # Make a plot - template custom, or interactive or flat
-    try:
-        return get_template_mod().linegraph(plotdata, pconfig)
-    except (AttributeError, TypeError):
```

Comment:
Huh, we just defaulted to flat plots without any log or anything? 😅

Reply:
Doesn't seem so? 🤔